frb_example/pure_dart/rust/src/api.rs

```diff
@@ -668,6 +669,26 @@ pub fn get_sum_struct() -> SumWith {
     SumWith { x: 21 }
 }
 
+pub struct Id(pub [u8; 32]);
+
+pub fn repeat_id(value: u8) -> Id {
+    Id([value; 32])
+}
+
+pub fn is_repeat_id(id: Id, repeat_value: u8) -> bool {
+    id.0.iter().all(|n| n.eq(&repeat_value))
+}
+
+pub struct Logs(pub [Log; 2]);
+
+pub fn logs(log_a: Log, log_b: Log) -> Logs {
+    Logs([log_a, log_b])
+}
+
+pub fn first_log(logs: Logs) -> Log {
```

Comment:
Btw, do we support things like: 

* `f(arg: [Sth; 2])`
* `f(arg: Box<[Sth; 2]>)` 

etc? If we support more kinds of things in this PR, would be great to add them to tests, such that we can verify it works correctly, and in the future nobody will accidentially break it.